source/user-manual/capabilities/file-integrity/fim-fields-rule-mapping.rst

```diff
@@ -0,0 +1,145 @@
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  group_id            |  audit.group.id             |  (Audit) group ID of the process                                |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  group_name          |  audit.group.name           |  (Audit) group name of the process                              |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  process_name        |  audit.process.name         |  (Audit) name of the process                                    |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  ppid                |  audit.process.ppid         |  (Audit) parent ID of the process                               |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  process_id          |  audit.process.id           |  (Audit) ID of the process                                      |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  effective_uid       |  audit.effective.user_id    |  (Audit) effective user ID of the process                       |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  effective_name      |  audit.effective_user.name  |  (Audit) effective user name of the process                     |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  parent_name         |  audit.process.parent_name  |  (Audit) parent process name                                    |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  cwd                 |  audit.process.cwd          |  (Audit) current work directory                                 |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  parent_cwd          |  audit.process.parent_cwd   |  (Audit) parent process's current work directory                |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  audit_uid           |  audit.login_user.id        |  (Audit) User login ID used by Audit                            |
++----------------------+-----------------------------+-----------------------------------------------------------------+
+|  audit_name          |  audit.login_user.name      |  (Audit) User name used by Audit                                |
```

Comment:
The audit fields give information about the user/process that caused changes that ultimately resulted in the current event. We should rephrase these fields so that we explicitly say the information is about the audit event that caused the FIM event to be triggered.

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3913/commits/d5ce0fcadd5abf8bbb40c82c8ccbba1b333c2572)

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3913/commits/6f538ba0f29e9eb74ddfc9e363f4de470e21bb84)